frontend/src/components/maps/leaflet/InfoSlideOut/InfoContent.tsx

```diff
@@ -80,6 +82,14 @@ export const InfoContent: React.FC<IInfoContent> = ({
   const isParcel =
     propertyTypeId !== null &&
     [PropertyTypes.PARCEL, PropertyTypes.SUBDIVISION].includes(propertyTypeId);
+
+  const lookupCodes = useCodeLookups();
+  const agencies = lookupCodes.getByType(API.AGENCY_CODE_SET_NAME);
+  const getFullAgencyName = (agencyCode?: string) => {
+    const agencyItem = agencies.find(listItem => listItem.code === agencyCode);
+    return agencyItem?.name;
```

Comment:
it may make sense to fall back on the code if the name search fails/returns a null name.